packages/cloud-functions/README.md

```diff
@@ -0,0 +1,13 @@
+# @uma/merkle-distributor
+
+This package contains a number of serverless functions used to wrap core UMA logic in a serverless consumable format.
+
+## Key packages included:
```

Comment:
Do you want these packages to be first-class citizens within our lerna setup? Or do you want lerna to effectively ignore them?

Reply:
Ideally, I think Lerna should ignore them. these are meant to be really self standing with minimal code within them. Right now there is a separate .gitignore that skips their `yarn.lock` files. Do you agree? 

My main reasoning is people should not really install/deploy/use the packages on their own outside of serveless and so having it in Lerna is not really any gain.

Reply:
I generally agree with the sentiment, but I want to make sure that this doesn't cause any unintended side effects. Some things I'm a bit worried about:
- `yarn` at root will not install their packages. You will have to manually jump to each directory and install.
- Similarly `yarn test` will not automatically pick up the `test` script in these packages if it exists. So we would either need to manually add the test command somewhere or not run tests in these packages.
- If these are implemented in typescript, the same thing would apply. Any calls to `yarn qbuild` or `yarn build` would not automatically hit these packages.

We can include things in lerna/yarn workspaces and _not_ publish them. The voter-dapp works this way. We just need to set it as private in the package.json and it will never be published.

Reply:
Just tried this out locally. All you should need to do to have lerna and yarn pick up these new packages is change the root level package.json:
```
  "workspaces": [
    "packages/**"
  ]
```

and lerna.json:
```
  "packages": ["packages/**"],
```

Reply:
nice find! I've made the updates :) thanks for the input!